core/io/resource.h

```diff
@@ -83,11 +83,12 @@ class Resource : public RefCounted {
 public:
 	static Node *(*_get_local_scene_func)(); //used by editor
 	static void (*_update_configuration_warning)(); //used by editor
+	static Ref<Resource> get_remap_resource(const Ref<Resource> &p_resource, HashMap<Ref<Resource>, Ref<Resource>> p_remap_cache, const Ref<Resource> p_fallback, Node *p_for_scene);
```

Comment:
Any reason `p_fallback` is not passed by reference yet is const?

Reply:
It was omitted before, but it has been corrected.